setup.md

```diff
@@ -0,0 +1,334 @@
+
+Click on it and then click on "Download ZIP".
+
+![download zip](./images/main-setup/oxcompbio-zipdownload.png)
+
+This will download a copy of the course materials to your computer. Then simply
+move the zip file to some convenient folder (ideally easily accessible through your
+terminal), and extract it (macOS: double click, windows: right-click and extract all).
+
+This will create a folder named `OxCompBio`.
+
+### Not recommended: Git clone 
+This is not a recommended option, but can be useful if you are using linux/macOS
+and are already quite familiar with `git`.
+
+Simply do:
+
+```
+git clone https://github.com/bigginlab/OxCompBio.git
+```
+
+In a convenient place for you to place the course materials. This will download
+everything automatically to a local folder named `OxCompBio`.
+
+
+## 4. [Jupyter notebooks]
+
+The tutorials rely on the use of [Jupyter notebooks] to provide an interactive
+experience, particularly when dealing with python code. The install
+instructions for each tutorial will install a copy of the jupyter notebook
+interface where required. We recommend that you start by setting up the
+[python tutorial] and then look at some of the following resources on how to
+use notebooks:
+
+- [The jupyter notebook beginners guide](https://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/execute.html)
+- [Codeacademy's jupyter notebook tutorial](https://www.codecademy.com/articles/how-to-use-jupyter-notebooks)
+- [Quick introduction to jupyter notebooks](https://www.youtube.com/watch?v=jZ952vChhuI) (youtube video)
+
+
+Briefly, one usually starts a jupyter notebook session by opening a terminal,
+navigating to the folder where `ipynb` (notebook) files are present, and
+typing the following:
+
+```
+jupyter notebook
+```
+
+This should automatically open an interactive session in your browser.
+
+
+![jupyter main session](./images/main-setup/jupyter-main-window.png)
+
+You can then click on the `ipynb` (notebook) file to start it.
+
+Jupyter notebooks are composed of cells, some of which contain normal text
+(with a white background), and code blocks (shaded in grey).
+
+![jupyter cells](./images/main-setup/jupyter-cells.png)
+
+Code blocks are meant to be interactive, you can type in them and press
+`shift+Enter` to run the contents of the cells.
+
+You can exit a notebook by clicking "File"->"Close and Halt"
+
+![jupyter close](./images/main-setup/jupyter-close.png)
+
+Sometimes when too many things are running, notebooks can start to
+behave erratically, when this happens clicking on the "interrupt" button
+may help:
+
+![jupyter-interrupt](./images/main-setup/jupyter-interrupt.png)
+
+Alternatively, using one of the "Kernel"->"restart" options can be very useful.
+Particularly, the "Restart & Clear Output" can be useful in cleaning up the notebook's
+outputs and starting again:
+
+![jupyter-restart](./images/main-setup/jupyter-restart.png)
+
```

Comment:
Do we need a comment about non-linear execution and the `Restart & Run all` or is it overkill?

Reply:
Could do, any suggestions on text (any handwavy blurb might be useful, I've kinda ran out of ideas 😅 )

Reply:
```suggestion
**Note**: In a Jupyter notebook you can run cells in any order you like. This might create problems in the long run, therefore we suggest to run the cells sequentially (from top to bottom). If you start to get strange results within a Jupyter notebook, you can use "Kernel"->"Restart & Run all" to run clean everything up and re-run the notebook from scratch.
```